.ruby-version

```diff
@@ -0,0 +1 @@
+2.2.4
```

Comment:
Can we not have this file please? If you need it locally, please feel free to .gitignore it